roles/icecast/templates/icecast.xml.j2

```diff
@@ -1,40 +1,40 @@
-<icecast>
-    <limits>
-        <clients>{{ icecast_clients|default(100) }}</clients>

```

Comment:
The reason that these were this way is due to the way ansible overrides group/host and roles variables. It doesn't merge the hash, but replaces it. 

http://docs.ansible.com/playbooks_variables.html#variable-precedence-where-should-i-put-a-variable
and
http://stackoverflow.com/questions/25129728/ansible-override-single-dictionary-key